t/t3428-rebase-edit-todo.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+test_description='rebase should reread the todo file if an exec modifies it'
+
+. ./test-lib.sh
+
+todo=$(git rev-parse --git-dir)/rebase-merge/git-rebase-todo
```

Comment:
BTW if that is the case here, i.e. if `todo=.git/rebase-merge/git-rebase-todo`, then it would make more sense to use that hard-coded assignment here. Or maybe even avoid the assignment and use the hard-coded path in the test case below.